logger.go

```diff
@@ -0,0 +1,7 @@
+package eventloghook
+
+type logger interface {
```

Comment:
pretty much not something most people would want to do but generally it would be good to export this interface so someone can mock it.

Otherwise this is a great improvement, I had considered adding the go.mod etc but didnt want to rock the boat :)